src/detect-snmp-community.c

```diff
@@ -101,7 +101,7 @@ static InspectionBuffer *GetData(DetectEngineThreadCtx *det_ctx,
         uint32_t data_len = 0;
         uint8_t *data = NULL;
 
-        rs_snmp_tx_get_community(txv, (uint8_t **)&data, &data_len);
+        rs_snmp_tx_get_community(txv, (const uint8_t **)&data, &data_len);
```

Comment:
I think this can be fixed more cleanly: by making data ```const uint8_t *```

Reply:
This one is no longer needed following a `autogen.sh; ./configure; make clean;make`.